### ISSUE TITLE
Handcuff Lottery

### ISSUE THREAD

**Issue by beaubreedlove** [beaubreedlove/FantasyFootball#7]
# tl;dr

- There's a strategy that might seem sketchy at first glance, but I would argue isn't against any rules and it adds fun to the game.
- The strategy is called the Handcuff Lottery.
- I'd like to allow the Handcuff Lottery to take place by not making any rules to disallow it.

To better understand it, start with these definitions:

## Definitions:

**Handcuff** -- A handcuff is any player whose value for any given week would increase significantly if some undetermined event (called the Handcuff Event from now on) happened vs. if that event didn't happen. The most common example of a Handcuff Event is the injury of a star player that is one string ahead of the handcuff. Other less common examples include players effected by court cases (think Zeke), weather events (the Hurricane in Miami last year), NFL trades, and other dramatic distractions from NFL players playing football. The value of a Handcuff is based partially on the chance that the Handcuff Event occurs, partially on the expected value of the Handcuff player after the Handcuff event occurs and partially on other factors.

**Handcuff Lottery** -- The process of picking up Handcuffs at a time slightly before a Handcuff Event is likely to occur (example: just before a game if the handcuff events is the injury of some player, or just before a court case or some other expected news for other potential handcuff players) and then subsequently dropping them after the time that the event was likely to occur (i.e. the game) for the next best available Handcuff on the FA.

## Advantages to Handcuff Lottery strategy

- Managers can optimize their teams for the greatest total probabilistic value of player on their team. If a player's probabilistic value decrease due to some event not happening, they are worthy of dropping for another player.

## Disadvantage to the Handcuff Lottery strategy

- There are only a small finite number of players on any given week whose actual value plus potential handcuff value times the chance that the handcuff event happens is more than the value of the worst player on any given team. This results in very few options (Lottery Tickets) for the Handcuff Lottery. Just the possibility that any other team may be playing the Handcuff Lottery at any given time, adds cost and risk to playing the Handcuff Lottery.

## Why I don't consider this Waiver Abuse

See: [Waiver Abuse](https://github.com/beaubreedlove/FantasyFootball/issues/6)

I think a key component to Waiver Abuse is causing some player to move to the Waiver Wire (WW) without some cost to the player who made the move. The only players who end up on the WW as a result of managers playing the Handcuff Lottery are players who would have ended up on the WW anyway. The most common use of the Handcuff Lottery is to drop players who have already played for the week or who are near the end of a game. In both of these cases the rule on Yahoo is that they players are moved to WW until Tuesday -- an effect that is no worse than being dropped from any given team.

## Why we should allow it

Since there's something to be gained for playing the Handcuff Lottery, but the total value any team gain acquire is reduced as more people play, there are tough decisions that managers need to make at any given time to determine whether to play the Handcuff Lottery. There is some psychology involved and ultimately, there's no move that is always better than the alternatives. I think this makes good strategy and I'd like to keep this dynamic in the league.
